core/src/main/scala/kafka/consumer/ZookeeperConsumerConnector.scala

```diff
@@ -778,9 +775,8 @@ private[kafka] class ZookeeperConsumerConnector(val config: ConsumerConfig,
                                        messageStreams: Map[String,List[KafkaStream[_,_]]],
                                        queuesToBeCleared: Iterable[BlockingQueue[FetchedDataChunk]]) {
       val allPartitionInfos = topicRegistry.values.map(p => p.values).flatten
-      fetcher match {
-        case Some(f) =>
-          f.stopConnections
+      fetcher.foreach { f =>
+          f.stopConnections()
```

Comment:
Generally, you have to fix the indent in every case where you replace a pattern match with `foreach`.